### DISCUSSION TITLE
Feature Request: Team Pick List Change

### DISCUSSION THREAD

**Discussion by CrankyLandlord** [LBPUnion/ProjectLighthouse#976] [Ideas]
As it stands, the Team Pick category will currently send whichever Team Pick is most recently updated to the top. This makes it hard to consistently feature new Picks: if a new level or levels are picked, a user who recieved a Team Pick in the past could simply update their level and reappear at the top of the list. This is pretty easily exploitable

**Comment by sudokoko** [comment]
Hello there, thank you for opening this discussion!

This has been an issue for some time now, however I believe we've not actioned upon it because adding the true original upload date would require a database migration which would not be able to account for the original upload dates. The best we could do is pull in the last updated timestamp and use that as the "original" upload date.

Calling upon @Slendy to correct me here if I'm wrong.

This could ultimately be reviewed again another time but it doesn't seem viable at this point. If a user is abusing their team pick status, it's ultimately up to the instance administrator(s) to take action upon this. In the context of Beacon, feel free to ask the Starguard Captain about it.

**Reply by Slendy** [reply]
We could change the TeamPick field to be a timestamp rather than a Bool so that team picked levels would show up in the order that they were picked. All previously team picked levels would have to be re-picked though to have the right order.

**Reply by sudokoko** [reply]
We could. If you feel this would work, feel free to move this to an issue

**Reply by CrankyLandlord** [reply]
That would be super helpful. Beacon is going to start  doing pretty regularly scheduled TP's again--there are a lot of barely played but terrific levels to TP--and it'd be nice to be able to consistenly feature them in-game without possibility of users accidentally overriding top spots

**Comment by sudokoko** [comment]
#980 has been merged into main.
